react/official/sidemenu/src/components/Menu.tsx

```diff
@@ -87,7 +87,7 @@ const Menu: React.FC = () => {
           <IonListHeader>Labels</IonListHeader>
           {labels.map((label, index) => (
             <IonItem lines="none" key={index}>
-              <IonIcon slot="start" icon={bookmarkOutline} />
+              <IonIcon aria-hidden={true} slot="start" icon={bookmarkOutline} />
```

Comment:
```suggestion
              <IonIcon aria-hidden="true" slot="start" icon={bookmarkOutline} />
```

`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.